docs/source/conceptual/visual_language_models.md

```diff
@@ -0,0 +1,224 @@
+# Vision Language Models (VLM)
+
+## What is VLM?
+
+Visual Language Model (VLM) are models that consume both visual and textual inputs to generate text.
+
+These models are trained on multimodal data, which includes both images and text.
+
+VLMs can be used for a variety of tasks, such as image captioning, visual question answering, and more.
```

Comment:
```suggestion
VLMs can be used for a variety of tasks, such as image captioning, visual question answering, and more.
```
these are super common ones that can also be done through models like BLIP that is not often what we mean here with VLMs (models with longer context) so we should add stuff like "multimodal chat" or "visual reasoning" which models like LLaVA is very good at

Reply:
mentioned right below the intro